### ISSUE TITLE
Use of theme causes frame to immediately close

### ISSUE THREAD

**Issue by alyms108** [catppuccin/emacs#106]
I downloaded the theme file to my `.doom.d/themes/` directory and set up my `config.el` as instructed for Doom, but when I restarted the emacs server and tried to open a client frame, the frame only stayed open a few seconds before closing. This happened both when when opening a new client in the terminal and when opening a new gui client. I'm running Manjaro with kernel 5.10 with Doom Emacs. I have emacs running as a daemon via systemctl and connect to it with `emacsclient`.

**Comment by KaranAhlawat** [comment]
Can you run a `emacs --debug-init` to find out any possible errors you may be running into?

**Comment by alyms108** [comment]
I ran the debugger and the first thing that stood out was seeing `<DOCTYPE!` in some of the output. When I checked the actual contents of the theme file, I realized I made a serious error. I thought I'd downloaded the actual theme file (I used `wget`), but instead I grabbed the *web page* that the file was displayed on. That's probably what kept tripping emacs up. I went and downloaded the actual theme file (and inspected it to make sure I was getting the right thing), and everything works as it should. Apologies, I really should have verified the download the first time around.
